### ISSUE TITLE
job_received events can be incorrect

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieGrader#9]
Following up on the issue reported in #7, this happens when the commits in that PR are merged. The test is:

1. Delete all existing `grader-dev` instances.
2. Create a new one using `grader.template` and this `send-received-to-webhook` branch.
3. Open two browser windows on the instructor question page for `fibonacciUpload`, pointing to different variants.
4. Upload the `fib_error_timeout_list` code in both windows, but don't click grade yet.
5. Click grade in both windows at almost the same time.

The PL logs are below (with unneeded junk deleted). Observe that multiple `job_received` events are sent to PL for `job_id = 297`.

```
{"timestamp":"2018-02-15T04:21:08.131Z","ip":"::1","forwardedIP":"18.219.55.242","authn_user_id":null,"authn_user_uid":null,"method":"POST","path":"/pl/webhooks/grading","params":{},"body":{"event":"job_received","job_id":297,"data":{"received_time":"2018-02-15T04:21:08.431Z"}},"response_id":"4615217a-293e-4bd5-8ad4-343e7c237bc5","level":"verbose","message":"request"}
{"timestamp":"2018-02-15T04:21:08.153Z","ip":"::1","forwardedIP":"18.219.55.242","authn_user_id":null,"authn_user_uid":null,"method":"POST","path":"/pl/webhooks/grading","params":{},"body":{"event":"job_received","job_id":298,"data":{"received_time":"2018-02-15T04:21:08.493Z"}},"response_id":"d6059b3c-c528-4b77-948f-d641e433adff","level":"verbose","message":"request"}
{"timestamp":"2018-02-15T04:22:08.063Z","ip":"::1","forwardedIP":"18.219.55.242","authn_user_id":null,"authn_user_uid":null,"method":"POST","path":"/pl/webhooks/grading","params":{},"body":{"event":"job_received","job_id":297,"data":{"received_time":"2018-02-15T04:22:08.015Z"}},"response_id":"4c99ed7d-84d3-4808-9d19-c17682e5dc4a","level":"verbose","message":"request"}
{"timestamp":"2018-02-15T04:23:24.985Z","ip":"::1","forwardedIP":"18.219.55.242","authn_user_id":null,"authn_user_uid":null,"method":"POST","path":"/pl/webhooks/grading","params":{},"body":{"data":{"start_time":"2018-02-15T04:22:54.335Z","timedOut":true,"end_time":"2018-02-15T04:23:24.496Z","succeeded":false,"message":"Grading timed out after 30 seconds.","results":null,"job_id":297,"received_time":"2018-02-15T04:22:08.015Z"},"event":"grading_result","job_id":297},"response_id":"97a0e44b-7c65-489f-b61c-15f76124e54a","level":"verbose","message":"request"}
{"timestamp":"2018-02-15T04:23:25.070Z","ip":"::1","forwardedIP":"18.219.55.242","authn_user_id":null,"authn_user_uid":null,"method":"POST","path":"/pl/webhooks/grading","params":{},"body":{"event":"job_received","job_id":298,"data":{"received_time":"2018-02-15T04:23:25.051Z"}},"response_id":"a4217db4-0380-49dc-ae47-04ff0c84202e","level":"verbose","message":"request"}
{"timestamp":"2018-02-15T04:23:56.216Z","ip":"::1","forwardedIP":"18.219.55.242","authn_user_id":null,"authn_user_uid":null,"method":"POST","path":"/pl/webhooks/grading","params":{},"body":{"data":{"start_time":"2018-02-15T04:23:25.612Z","timedOut":true,"end_time":"2018-02-15T04:23:55.776Z","succeeded":false,"message":"Grading timed out after 30 seconds.","results":null,"job_id":298,"received_time":"2018-02-15T04:23:25.051Z"},"event":"grading_result","job_id":298},"response_id":"00e32e2a-92d8-4979-a5b6-cf2db81f76f5","level":"verbose","message":"request"}
```

**Comment by mwest1066** [comment]
In fact, looking at the above log, we get two `job_received` events for each job, which is what causes the problem.

**Comment by mwest1066** [comment]
For now I've reverted 51b7347850739bb452a3802fb483295711154c79

**Comment by mwest1066** [comment]
Resolved by #41 with switch to SQS for `job_received` reporting
